src/main/java/propertyFiles/Bundle.properties

```diff
@@ -2948,3 +2948,7 @@ publishDatasetCommand.pidNotReserved=Cannot publish dataset because its persiste
 api.errors.invalidApiToken=Invalid API token.
 api.ldninbox.citation.alert={0},<br><br> The {1} has just been notified that the {2}, <a href=''{3}''>{3}</a>, cites "<a href=''{4}/dataset.xhtml?persistentId={5}''>{6}</a>" in this repository.
 api.ldninbox.citation.subject={0}: A Dataset Citation has been reported!
+
+#Info.java
+info.api.exception.invalid.format=Invalid format {0}, currently supported formats are YAML and JSON.
+info.api.exception=Supported format definition not found.
```

Comment:
Can we add "openapi" to the key for these?

Reply:
`info.api.openapi.exception.invalid.format` and `info.api.openapi.exception`?